apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -1036,7 +1036,7 @@ def generate(el, data):
             "offset_backward": offset_backward,
             "originY": "center",
             "selectable": drawing_defaults["selectable"],
-            "clockwiseDirection": clockwise_direction,
+            "clockwise-direction": clockwise_direction,
```

Comment:
Good catch regarding the inconsistent name of the option. I believe, for some consistency, it would make more sense to change it in the other direction, i.e., change `clockwise-direction` to `clockwiseDirection` instead of the other way around.